src/features/courses/course-repository.ts

```diff
@@ -209,6 +209,16 @@ class CourseRepository {
                 id: options.id,
                 active: true
             },
+            include: [
```

Comment:
Should this include be a passed in option so it is not always executed?

Reply:
I don't think so. Would we add a new flag for 'includeExams'? I don't really see what the benefit to that would be.

Reply:
less queries if you were fetching the topic for some other reason

Reply:
we'd have to otherwise check that a topic is an exam anyhow - I certainly make use of this include being here when I request a topic.